throughput/tracker.go

```diff
@@ -48,30 +48,36 @@ func (t *tracker) logResult(txID ids.ID, result *chain.Result) {
 func (t *tracker) logState(ctx context.Context, cli *jsonrpc.JSONRPCClient) {
 	// Log stats
 	tick := time.NewTicker(1 * time.Second) // ensure no duplicates created
-	var psent int64
+	var (
+		prevSent int64
+		prevTime = time.Now()
+	)
 	go func() {
 		defer tick.Stop()
 		for {
 			select {
 			case <-tick.C:
-				current := t.sent.Load()
 				t.l.Lock()
 				if t.totalTxs > 0 {
 					unitPrices, err := cli.UnitPrices(ctx, false)
 					if err != nil {
 						continue
 					}
+					currSent := t.sent.Load()
+					currTime := time.Now()
+					diff := min(currTime.Sub(prevTime).Seconds(), 1)
```

Comment:
is it possible for less than 1s to elapse, such that taking the min with 1s will produce an incorrect result